src/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "example-app",
+    "private": true,
+    "type": "module",
+    "scripts": {
+        "dev": "vite",
+        "build": "vite build",
+        "lint": "eslint .",
+        "lintf": "eslint . --fix"
```

Comment:
We can add [type checking](https://www.npmjs.com/package/vue-tsc) if we are already using TypeScript. `"tsc": "vue-tsc"`

Reply:
added